lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
I think the default value is only present for the GSS keytab setting, but dns_key and probably stuff in other modules (Puppet?) etc may not have defaults, so might they have nil values too?

Remind me, what's it needed for?